src/opm/parser/eclipse/EclipseState/Schedule/Action/PyAction.cpp

```diff
@@ -28,12 +29,45 @@ using dict = int;
 #endif
 
 
+#include <opm/parser/eclipse/Utility/String.hpp>
+#include <opm/common/utility/FileSystem.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/Action/PyAction.hpp>
 
 namespace Opm {
+namespace Action {
 
-PyAction::PyAction(const std::string& code_arg) :
-    input_code(code_arg),
+PyAction::RunCount PyAction::from_string(std::string run_count) {
+    run_count = uppercase(run_count);
+
+    if (run_count == "SINGLE")
+        return RunCount::single;
+
+    if (run_count == "UNLIMITED")
+        return RunCount::unlimited;
+
+    if (run_count == "FIRST_TRUE")
+        return RunCount::first_true;
+
+    throw std::invalid_argument("RunCount string: " + run_count + " not recognized ");
+}
+
+
+std::string PyAction::load(const std::string& input_path, const std::string& fname) {
+    namespace fs = Opm::filesystem;
+    fs::path code_path = fs::path(input_path) / fs::path(fname);
+    if (fs::exists(code_path)) {
+        std::ifstream ifs(code_path.c_str(), std::ios::in);
```

Comment:
You're supposed to be able to construct a `basic_ifstream` object directly from an `fs::path` object.  Did the following not work for you?
```C++
std::ifstream ifs(code_path);
```

Reply:
I did not know it was possible - I will update.